source/postprocess/boundary_velocity_residual_statistics.cc

```diff
@@ -148,7 +192,15 @@ namespace aspect
                 for (unsigned int q=0; q<fe_face_values.n_quadrature_points; ++q)
                   {
                     const Point<dim> point_at_surface = fe_face_values.quadrature_point(q);
-                    // Extract data velocity.
+                    // Extract data velocity or strain rate.
+
+                    if (use_surface_strain_rate_data)
+                      {
+                        std::vector<Vector<double> > data_surface_strain_rate = get_data_surface_strain_rate(point_at_surface);
```

Comment:
this would be now
```suggestion
                        Tensor<2,dim-1> data_surface_strain_rate = get_data_surface_strain_rate(point_at_surface);
```